include/aktualizr-lite/aklite_client_ext.h

```diff
@@ -0,0 +1,64 @@
+#ifndef AKTUALIZR_LITE_CLIENT_EXT_H_
+#define AKTUALIZR_LITE_CLIENT_EXT_H_
+
+#include <cstdlib>
+#include <string>
+
+#include "json/json.h"
+
+#include "aktualizr-lite/tuf/tuf.h"
+
+#include "aktualizr-lite/api.h"
+
+class GetTargetToInstallResult {
+ public:
+  //
+  enum class Status {
+    // First block must match CheckInResult::Status
+    Ok = 0,    // check-in was good
```

Comment:
Maybe, doing `Ok = static_cast<int>(CheckInResult::Status::Ok)` will be more reliable.

Reply:
Done.